build

```diff
@@ -490,6 +495,7 @@ setupccache() {
 	chown -R "$ABUILD_UID:$ABUILD_GID" "$BUILD_ROOT/.ccache"
 	echo "export CCACHE_DIR=/.ccache" > "$BUILD_ROOT"/etc/profile.d/build_ccache.sh
 	echo 'export PATH=/var/lib/build/ccache/bin:$PATH' >> "$BUILD_ROOT"/etc/profile.d/build_ccache.sh
+	tar -zxf "$BUILD_ROOT/.build.oldpackages/_ccache.tar.gz" -C /.ccache/
```

Comment:
don't we need an existens check here?

(We might do not have a former build result or the method is used in old scenario)

Reply:
the deliberately left it without check: 

- tar will just throw an error into the log file, indicating an error condition ($ccache is set but the archive is not there). There is no sure shot way to know if this the first build with --ccache (in which case, ofcouse there is no archive) but subsequent builds should have this archive present and if this archive is absent that is an error condition.

I think its an overkill to fail the entire build just because ccache is no present.

Reply:
Also if the path with  --pkg-ccahe is not correct (typo or any other reason), tar will indicate that archive is not present. As will also be indicated by rsync